src/wallet/api.rs

```diff
@@ -1332,7 +1332,9 @@ impl Wallet {
             change_cost: cost_of_change,
             avg_input_weight,
             avg_output_weight,
-            min_change_value: 100,
+            // The purpose of keeping it hardcoded is 2 fold. It gives a minimum threshold for Privacy Regular Swaps and solves for the check in SpendAPI against minimal dust and subtracting by fee_wchange - since our current coinselection doesn't handle multiple change outputs.
+            // Instead of hardcoding it, we can definitely do it in a smarter manner.
```

Comment:
Shorten this comment and explain why 294 * 5 is selected as. Why not any other values?

What is "Privacy Regular Swaps"? Give it a better name. Simplify the comment to explain the effect of this minimal change amount in coinselection.

Add a TODO mentioning it to be improved later.